Autopilot/AttitudeManager/StateMachine.c

```diff
@@ -24,14 +23,10 @@ int downlinkP1Timer = 0;
 int downlinkP2Timer = 0;
 int imuTimer = 0;
 char AMUpdate = 0;
+char flightUpdate = 0;
```

Comment:
i think having verbs first makes it sound more natural. like if(updateFlight) vs if(flightUpdate)